CorsixTH/Lua/world.lua

```diff
@@ -1379,11 +1379,11 @@ end
 
 -- PRIVATE method to generate the next VIP date
 function World:_generateNextVipDate()
-  -- Support standard values for mean and variance
-  local mean = 180
-  local variance = 30
+  -- Support standard values for mean and bound
+  local mean, bound = self.map.level_config.gbv.MayorLaunch, 30
```

Comment:
I think 10 days between VIP visits would be nice to keep the user on its toes :p 
Your bound may now cause VIPs traveling in time to earn more money in the past.